ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -128,9 +128,9 @@ class LeftNavMenuViewController: UIViewController {
 
     private var persona = MSFPersonaView()
 
-    private var leftNavMenuList = MSFList()
+    private var leftNavMenuList = FluentList()
 
-    private var statusCell: MSFListCellState?
+    private var statusCell: FluentListCellState?
```

Comment:
I don't think MSFListCellState needed to change to FluentListCellState. If you look at the vNext button, we have FluentButton and MSFButtonStateImpl. We also have FluentButtonStyle, but maybe there was a naming collision on ButtonStyle? Anyway, I think most of the names were okay as MSF*

Reply:
Hm okay, so just the ones involving the big List, and excluding anything stemming from List? (Cell, Header, etc)

Reply:
Yeah, that sounds right to me. The actual view / control shouldn't be MSF, but everything else is fine. So, in this case, I think we should have MSFList (currently FluentList) and FluentList (currently FluentListView), since what's currently names FluentListView is the actual SwiftUI control, while what's currently FluentList is just the hosting container. Relatedly, MSFList should get moved to its own file to match the other controls.